source/uwp/SharedRenderer/lib/AdaptiveImageRenderer.cpp

```diff
@@ -289,74 +289,78 @@ namespace AdaptiveCards::Rendering::Xaml_Rendering
             adaptiveCardElement, selectAction, renderContext, frameworkElement, XamlHelpers::SupportsInteractivity(hostConfig), true);
     }
 
-    template<typename TElement>
-    winrt::ImageSource XamlBuilder::SetImageOnUIElement(winrt::Uri const& imageUrl,
-                                                        winrt::AdaptiveCardResourceResolvers const& resolvers,
-                                                        ImageProperties<TElement> const& imgProperties)
+    winrt::Windows::Foundation::Size XamlBuilder::ParseSizeOfSVGImageFromString(winrt::hstring const& content)
     {
-        bool mustHideElement = true;
+        // Parse the size from the XamlDocument
+        winrt::XmlDocument xmlDoc;
 
-        // Get the image url scheme
-        winrt::hstring schemeName = imageUrl.SchemeName();
+        xmlDoc.LoadXml(content);
 
-        // Get the resolver for the image
-        if (resolvers)
+        if (xmlDoc)
         {
-            auto resolver = resolvers.Get(schemeName);
-            // If we have a resolver
-            if (resolver)
+            auto rootElement = xmlDoc.DocumentElement();
+
+            // Root element must be an SVG
+            if (winrt::operator==(rootElement.NodeName(), L"svg"))
```

Comment:
this feels a bit odd -- we don't normally see explicit invocation of operators...

Reply:
thanks; I missed it.